packages/core/lib/commands/call/meta.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  command: "call",
+  description: "Call read-only contract function with arguments",
+  builder: {},
+  help: {
+    usage:
+      "truffle call <contract-name> <function-name> [<function-args...>] [--network=<network-name>",
```

Comment:
Although this just says `<contract-name>`, an initial release of this should include support for `--fetch-external`, and thus you should be allowed to specify either a contract name or an address.

Reply:
and for `<function-name>`, this should also allow the raw signature (e.g. `truffle call MyContract 'myFunction(string,uint)' ...`

the reason for this is: consider a contract that defines function `f` with two overloads (`f(string)` and `f(uint256)`). Today, if you pass a numeric string to the encoder, then the encoder's overload resolution system will always favor the latter overload, thus there's no good way currently to explicitly specify to use the string overload.

in addition, this situation probably even warrants a warning of some kind (e.g. "We notice you're calling a weirdly overloaded function... you might want to specify explicitly which overload you want")